llms/tests/test_lora.py

```diff
@@ -60,6 +65,22 @@ def check_config(params):
         params["keys"] = ["self_attn.k_proj"]
         check_config(params)
 
+        params["keys"] = ["lm_head"]
```

Comment:
The full path the layer that is being targeted for both the LM head (if the model has one) and the embeddings. @awni